creator-node/src/routes/nodeSync.js

```diff
@@ -32,126 +34,148 @@ module.exports = function (app) {
+              await Promise.all([
+                models.AudiusUser.findAll({
+                  where: {
+                    cnodeUserUUID: cnodeUserUUIDs,
+                    clock: {
+                      [models.Sequelize.Op.gte]: requestedClockRangeMin,
+                      [models.Sequelize.Op.lte]: requestedClockRangeMax
+                    }
+                  },
+                  order: [['clock', 'ASC']],
+                  transaction,
+                  raw: true
+                }),
+                models.Track.findAll({
+                  where: {
+                    cnodeUserUUID: cnodeUserUUIDs,
+                    clock: {
+                      [models.Sequelize.Op.gte]: requestedClockRangeMin,
+                      [models.Sequelize.Op.lte]: requestedClockRangeMax
+                    }
+                  },
+                  order: [['clock', 'ASC']],
+                  transaction,
+                  raw: true
+                }),
+                models.File.findAll({
+                  where: {
+                    cnodeUserUUID: cnodeUserUUIDs,
+                    clock: {
+                      [models.Sequelize.Op.gte]: requestedClockRangeMin,
+                      [models.Sequelize.Op.lte]: requestedClockRangeMax
+                    }
+                  },
+                  order: [['clock', 'ASC']],
+                  transaction,
+                  raw: true
+                }),
+                models.ClockRecord.findAll({
+                  where: {
+                    cnodeUserUUID: cnodeUserUUIDs,
+                    clock: {
+                      [models.Sequelize.Op.gte]: requestedClockRangeMin,
+                      [models.Sequelize.Op.lte]: requestedClockRangeMax
+                    }
+                  },
+                  order: [['clock', 'ASC']],
+                  transaction,
+                  raw: true
+                })
+              ])
+
+            await transaction.commit()
+
+            /** Bundle all data into cnodeUser objects to maximize import speed. */
+
+            const cnodeUsersDict = {}
+            cnodeUsers.forEach((cnodeUser) => {
+              // Add cnodeUserUUID data fields
+              cnodeUser.audiusUsers = []
+              cnodeUser.tracks = []
+              cnodeUser.files = []
+              cnodeUser.clockRecords = []
+
+              cnodeUsersDict[cnodeUser.cnodeUserUUID] = cnodeUser
+              const curCnodeUserClockVal = cnodeUser.clock
+
+              // Validate clock values or throw and error
+              const maxClockRecordId = Math.max(
+                ...clockRecords.map((record) => record.clock)
+              )
+              if (cnodeUser.clock !== maxClockRecordId) {
+                throw new Error(
+                  `Cannot sync - exported data is not consistent. Exported max clock val = ${cnodeUser.clock} and exported max ClockRecord val ${maxClockRecordId}`
```

Comment:
can we change this err to say `Cannot export - data is not consistent. Exported max clock val = ${cnodeUser.clock} and exported max ClockRecord val ${maxClockRecordId}`. sync is what the secondary runs, this is the export 🙂